### PR TITLE
feat: improve visual affordance of horizontal tabs

### LINKED ISSUE #3299: Improve visual affordance of horizontal tabs. Database Info (light theme)
**Context**
A user reported that the horizontal tabs on the Database page in light theme were not perceived as interactive — they appeared static or non-switching. To address this, we’re starting with a quick visual fix: changing the background color of unselected tabs to improve perceived interactivity before testing further enhancements. 

[Figma Mockups](https://www.figma.com/design/bNbecZuPFDDrJ7ghaXZ2Fe/Tickets?node-id=12132-64063&t=C6Hl9ChSqXaBuh6L-4)
Design ticket: UI/UX — Quantitative and qualitative testing of Storage tab interface in light theme [#3227](https://github.com/ydb-platform/ydb-embedded-ui/issues/3227)

Image

**Objective**
Increase the visual distinction between selected and unselected tabs in the light theme, making it clearer that unselected tabs are clickable.

- Update the background color of unselected tabs in the light theme only.
- Remove shadow of unselected tabs in the light theme only.
- Keep all other tab properties unchanged (hover effects, selected state, cursor, borders, typography, etc.).

Image

### PR DESCRIPTION
**Pull request by kkdras**
Resolves #3299
[Stand](https://nda.ya.ru/t/v5sUMY9m7gpufq)
[Stand](https://nda.ya.ru/t/jfF3oiZE7gpuQN) bad conditions
[Stand](https://nda.ya.ru/t/FzCwHDzZ7gpunv) serverless

### Test Status: <span style="color: orange;">⚠️ FLAKY</span>
  📊 [Full Report](https://ydb-platform.github.io/ydb-embedded-ui/4065/?t=1782742234208)

  | Total | Passed | Failed | Flaky | Skipped |
  |:-----:|:------:|:------:|:-----:|:-------:|
  | 726 | 723 | 0 | 3 | 0 |

  
  <details>
  <summary>Test Changes Summary ✨3 </summary>

  #### ✨ New Tests (3)
1. Info metric tabs match visual baseline (tenant/diagnostics/tabs/info.test.ts)
2. Info serverless metric tabs match visual baseline (tenant/diagnostics/tabs/info.test.ts)
3. Info metric tabs keep the same width when active tab changes (tenant/diagnostics/tabs/info.test.ts)

  </details>

  ### Bundle Size: 🔽
  Current: 64.23 MB | Main: 64.24 MB
  Diff: 4.70 KB (-0.01%)

  ✅ Bundle size decreased.

  <details>
  <summary>ℹ️ CI Information</summary>

  - Test recordings for failed tests are available in the full report.
  - Bundle size is measured for the entire 'dist' directory.
  - 📊 indicates links to detailed reports.
  - 🔺 indicates increase, 🔽 decrease, and ✅ no change in bundle size.
  </details>

### CODE REVIEW

**Comment by kkdras** [comment]
@greptile review

**Comment by kkdras** [comment]
@codex review

**Comment by kkdras** [comment]
@codex review
@greptile review

File: src/containers/Tenant/Diagnostics/TenantOverview/TabCard/MetricTabCard.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+            <Flex direction="column" gap={0.5}>
+                <Flex alignItems="center" gap={1}>
+                    <Text variant="subheader-2" data-qa="tenant-metric-tab-title">
+                        {title}
+                    </Text>
+                    <Flex alignItems="center" gap={1}>
+                        <Text variant="body-1" color="secondary" data-qa="tenant-metric-tab-value">
+                            {value}
+                        </Text>
+                        <StatusIcon status={status} mode="icons" size="s" />
```
Comment (astandrik) [inline src/containers/Tenant/Diagnostics/TenantOverview/TabCard/MetricTabCard.tsx:31]: Problem: #3299 specifies the first-line status as icon + percentage value, but the card renders percentage first and the status icon second.

Repro:
1. Open the Database Info tab on this branch.
2. Check any metric tab first line.
3. Actual: `7% <icon>`. Expected from the issue: `<icon> 7%`.

Links:
- https://github.com/ydb-platform/ydb-embedded-ui/issues/3299 - source issue; the proposed card structure says `icon + percentage value`.

Please flip the icon/value order and update the visual snapshots.

File: src/containers/Tenant/Diagnostics/TenantOverview/MetricsTabs/metricTabPresentation.ts [hunk 2]
```diff
@@ -0,0 +1,71 @@
+import {EFlag} from '../../../../../types/api/enums';
+import {formatPercent} from '../../../../../utils/dataFormatters/dataFormatters';
+import {getMetricPercentPrecision} from '../../../../../utils/metrics';
+import {calculateProgressStatus} from '../../../../../utils/progress';
+import type {ProgressStatus} from '../../../../../utils/progress';
+
+const UNAVAILABLE_PERCENT_TEXT = 'N/A';
```
Comment (astandrik) [inline src/containers/Tenant/Diagnostics/TenantOverview/MetricsTabs/metricTabPresentation.ts]: Problem: the unavailable percent text is hardcoded as `N/A`, and the new test asserts that string. This keeps the empty metric state inconsistent with the shared UI placeholder.

Repro:
1. Render a metric tab with an absent/zero limit, e.g. `getUsageMetricTabPresentation({value: 10, limit: 0})`.
2. The tab shows `N/A` instead of the shared `EMPTY_DATA_PLACEHOLDER`.

Links:
- https://github.com/ydb-platform/ydb-embedded-ui/pull/4065#discussion_r3481981701 - earlier PR thread that raised the same placeholder issue.
- https://github.com/ydb-platform/ydb-embedded-ui/blob/main/AGENTS.md#data-edge-cases - repo rule for empty UI values.

Please use `EMPTY_DATA_PLACEHOLDER` here and update the test expectation.

File: src/containers/Tenant/Diagnostics/TenantOverview/MetricsTabs/MetricsTabs.tsx [hunk 3]
```diff
@@ -80,7 +79,6 @@ export function MetricsTabs({
     blobStorageStats,
     tabletStorageStats,
     networkUtilization,
-    networkThroughput,
     storageGroupsCount,
     controlPlaneNodesCount,
```
Comment (astandrik) [inline src/containers/Tenant/Diagnostics/TenantOverview/MetricsTabs/MetricsTabs.tsx]: Problem: `controlPlaneNodesCount` and `storageGroupsCount` are now dead props. The old node/group legends were removed, but the parent still computes/passes these values and the child prop types still declare them.

Repro:
1. Search this branch for `controlPlaneNodesCount` and `storageGroupsCount`.
2. They are computed/passed in `TenantOverview`/`MetricsTabs`, but `CpuTab` and `StorageTab` no longer read them.

Links:
- https://github.com/ydb-platform/ydb-embedded-ui/issues/3299 - source issue; the new card structure removes absolute values/old legends from these tabs.

Please remove the unused props and parent-side computations/passing.

**Review by astandrik — CHANGES_REQUESTED** [review]
Requesting changes for three small PR-scope issues: one acceptance mismatch, one empty-state consistency issue, and one dead prop cleanup.

File: src/containers/Tenant/Diagnostics/TenantOverview/TabCard/MetricTabCard.tsx (hunk 1 above)
Comment (kkdras) [reply src/containers/Tenant/Diagnostics/TenantOverview/TabCard/MetricTabCard.tsx:31]: This order is intentional. The latest Figma mocks were updated and now show the value first and the status icon after it, so the implementation follows the current design

File: src/containers/Tenant/Diagnostics/TenantOverview/MetricsTabs/MetricsTabs.tsx (hunk 3 above)
Comment (kkdras) [reply src/containers/Tenant/Diagnostics/TenantOverview/MetricsTabs/MetricsTabs.tsx]: Fixed: removed `controlPlaneNodesCount` and `storageGroupsCount` from `TenantOverview`, `MetricsTabs`, `CpuTab`, and `StorageTab`

File: src/containers/Tenant/Diagnostics/TenantOverview/MetricsTabs/metricTabPresentation.ts (hunk 2 above)
Comment (kkdras) [reply src/containers/Tenant/Diagnostics/TenantOverview/MetricsTabs/metricTabPresentation.ts]: I kept `N/A` here because this text comes from the current design for the metric tab unavailable state, but it would be cleaner to move it to i18n instead of keeping a local constant, so I extracted it there

**Review by astandrik — APPROVED** [review]
